models/webhook.go

```diff
@@ -482,6 +482,57 @@ func UpdateHookTask(t *HookTask) error {
 	return err
 }
 
+// PrepareWebHook adds special webhook to task queue for given payload.
+func PrepareWebHook(w *Webhook, repo *Repository, event HookEventType, p api.Payloader) error {
```

Comment:
Could we call it `PrepareWebhook` for consistency?

Reply:
Since the function named `PrepareWebHooks` for fire all webhooks of a repository or an orgnization, I named this funtion `PrepareWebHook` for one webhook is reasonable.

Reply:
@lunny That function is named `PrepareWebhooks`, not `PrepareWebHooks` (https://github.com/go-gitea/gitea/blob/master/models/webhook.go#L486)